products/browsersim-standalone/pom.xml

```diff
@@ -198,6 +232,8 @@
 									<zipfileset excludes="META-INF/*, about.html" src="${org.eclipse.jetty.websocket.servlet.file}"/>
 								</swtjar> 
 								<delete dir="target/application/temp"/>
+								<!-- JBIDE-18772 rename the default org.jboss.tools.browsersim-standalone-3.7.0-SNAPSHOT.zip to jbosstools-4.3.0.Beta2-browsersim-standalone-standalone.zip -->
+								<move file="${project.build.directory}/${project.artifactId}-${project.version}.zip" tofile="${project.build.directory}/jbosstools-${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}.${BUILD_ALIAS}-browsersim-standalone.zip"/>
```

Comment:
Yes, renaming/copying is to create the correct artifact by the name it'll be when we publish the snapshot, so that NO renaming is needed to stage or release.